ui/src/app/views/workspace-share/component.ts

```diff
@@ -52,11 +57,10 @@ export class WorkspaceShareComponent implements OnInit {
         this.loadingWorkspace = false;
         this.userEmail = profile.username;
       });
-    });
-  }
+    },
+    (error) => {
```

Comment:
Did you mean to ignore this error?

Reply:
I wasn't sure under which circumstances it might throw an error, but I wanted to put in a placeholder just in case.

Reply:
If you're not handling the error in this PR, please remove. Right now it's just a no-op so it's extraneous code.

Reply:
done